cloudbuild.yaml

```diff
@@ -10,8 +10,6 @@ steps:
         make test test_dataplex_metadata
         make test test_dataplex_metadata_uri_templates
         make test test_dataplex_dq_configs_cache
-        make test test_dataplex_integration_performance
-        make test test_dataplex_task
```

Comment:
Why do we remove these two tests?

Reply:
I will be adding them eventually, cloud build was giving timeout issues so removed them for now. I will create the separate PR for the same.

Reply:
The tests have been included back, as the timeout and the batches issue has been fixed after the rollback from dataproc team.